src/index.js

```diff
@@ -1,3 +1,7 @@
-import { exportFunction } from './second';
+import _ from './second';
```

Comment:
This is a great way to showcase that with default exports, you really can name them whatever you want--such as `_`.  Please note that this typically wouldn't be a proper name since `_` has been popularized by the [lodash](https://lodash.com/) library.

Reply:
I was trying it without this _, but was not working so found this solution. What could be the other way to call this function? I tried calling the function using function's name.

Reply:
It actually just depends on how you export the function.  [HERE](https://github.com/priyankacheema/project-setup/blob/dev/src/second.js#L1) you are saying `export default` and then declaring your function.  When you export using this syntax you are essentially saying, "It doesn't matter what any other file chooses to call you, you will always be bound to this function.  That's why in your other file, you are saying 
`import _ from './second';` the `_` refers to that function, but note that you can change the `_` to `jkfdakljfd` and it would work the same when you call it.

The other type is a `named export`.  An nice example of this can be found [HERE](https://github.com/parshanapally/project-setup/blob/master/src/second.js#L1). Note that the keyword `default` is not used. With this type of export, you are saying, "Any file that wants to import this piece of code, **must** use this functions name.  This is done by using curly braces and inside of them, putting the name(s) of the function(s) that you'd like to import from that file.  An example of this can be found [HERE](https://github.com/parshanapally/project-setup/blob/master/src/index.js#L1).